### PR TITLE
Support for reporting error message through pub/sub

### PR DESCRIPTION
**Pull request by nbali**

### CODE REVIEW

File: bqpump-core/src/main/java/ai/aliz/bqpump/bigquery/BigQueryUtil.java [hunk 1]
```diff
@@ -41,12 +43,12 @@ public void waitForJobToComplete(Job job) {
                 log.info("Job {} finished successfully", jobId);
             } else {
                 List<BigQueryError> executionErrors = completedJob.getStatus().getExecutionErrors();
-                executionErrors.forEach(e -> log.error("Error during job: {}", e));
-                throw new RuntimeException(String.format("Error during job (%s) execution, see the errors above", jobId));
+                executionErrors.forEach(e -> log.warn("Error during job: {}", e));
```
Comment (csabakassai) [inline bqpump-core/src/main/java/ai/aliz/bqpump/bigquery/BigQueryUtil.java]: It is an error, should be logged as error

**Comment by csabakassai** [comment]
Why do you need the error notification in Pub/Sub? A simple stackdriver alert is not enough?

**Comment by nbali** [comment]
"Why do you need the error notification in Pub/Sub? A simple stackdriver alert is not enough?"

I'm not the one needing it. The customers might need it. We should be able to show them some reason, why the file they have uploaded can't be "pumped" into BQ.

**Comment by nbali** [comment]
@csabakassai

**Pull request merged**